packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -166,6 +166,6 @@ describe('SearchWithin', function () {
     let group = getByRole('group');
 
     expect(searchfield).toHaveAttribute('aria-label', 'Aria Label');
-    expect(group).toHaveAttribute('aria-labelledby', 'Aria Label');
+    expect(group).toHaveAttribute('aria-labelledby', searchfield.id);
```

Comment:
Should the tests also check the default value for `aria-label`?

Reply:
Good idea, I'll add